### PR TITLE
Complete javascript-loops-practice exercise.

### PR DESCRIPTION
**Pull request by alexei-dayrit**
Screen Shot 2022-02-03 at 1 32 39 PM

### CODE REVIEW

File: javascript-loops-practice/get-indexes.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/* exported getIndexes */
+
+function getIndexes(array) {
+  var newArray = [];
+  for (var i = 1; i <= array.length; i++) {
+    newArray.push(i - 1);
```
Comment (codingcodymiller) [inline javascript-loops-practice/get-indexes.js:6]: Why not just start at `0` so you don't need to subtract?

File: javascript-loops-practice/add-suffix-to-all.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* exported addSuffixToAll */
+
+function addSuffixToAll(words, suffix) {
+  var array = [];
+  for (var i = 0; i < words.length; i++) {
+    array.push(words[i] + suffix);
+  } return array;
```
Comment (codingcodymiller) [inline javascript-loops-practice/add-suffix-to-all.js:7]: Please move `return` statements to their own line.

File: javascript-loops-practice/countdown.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/* exported countdown */
+
+function countdown(number) {
+  var array = [];
+  for (var i = 0; i <= number; i++) {
+    array.unshift(i);
+  } return array;
```
Comment (codingcodymiller) [inline javascript-loops-practice/countdown.js:7]: Please move `return` statements to their own line.

File: javascript-loops-practice/filter-out-strings.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+/* exported filterOutStrings */
+
+function filterOutStrings(values) {
+  var newArray = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] !== 'string') {
+      newArray.push(values[i]);
+    }
+  } return newArray;
```
Comment (codingcodymiller) [inline javascript-loops-practice/filter-out-strings.js:9]: Please move `return` statements to their own line.

File: javascript-loops-practice/find-index.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+/* exported findIndex */
+
+function findIndex(array, value) {
+  for (var i = 0; i < array.length; i++) {
+    if (value === array[i]) {
+      return i;
+    }
+  } return (-1);
```
Comment (codingcodymiller) [inline javascript-loops-practice/find-index.js:8]: Please move `return` statements to their own line.

File: javascript-loops-practice/includes-seven.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+/* exported includesSeven */
+
+function includesSeven(array) {
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === 7) {
+      return true;
+    }
+  } return false;
```
Comment (codingcodymiller) [inline javascript-loops-practice/includes-seven.js:8]: Please move `return` statements to their own line.

File: javascript-loops-practice/odd-or-even.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+function oddOrEven(numbers) {
+  var newArray = [];
+  for (var i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+      newArray.push('even');
+    } else {
+      newArray.push('odd');
+    }
+  } return newArray;
```
Comment (codingcodymiller) [inline javascript-loops-practice/odd-or-even.js:11]: Please move `return` statements to their own line.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by alexei-dayrit** [comment]
Screen Shot 2022-02-04 at 1 16 53 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**